mill-db-repo/src/main/java/org/duracloud/mill/db/repo/MillJpaRepoConfig.java

```diff
@@ -91,7 +91,11 @@ public BasicDataSource millRepoDataSource() {
         emf.setPackagesToScan("org.duracloud.mill");
 
         JpaConfigurationUtil.configureEntityManagerFactory(env,emf);
-
+        if(Boolean.parseBoolean(env.getProperty("generate.database", "false"))){
+            Properties properties = new Properties();
+            properties.setProperty("javax.persistence.schema-generation.database.action", "create");
```

Comment:
Ok, I see. I was reading it incorrectly. Perhaps pulling the env.getProperty() call up to a prior line would help to clarify. Up to you.